packages/ffe-grid-react/src/GridCol.js

```diff
@@ -65,10 +65,21 @@ const modifiers = props =>
         .map(key => `ffe-grid__col--${camelCaseToDashCase(key)}`)
         .join(' ');
 
-const backgroundClass = props =>
-    props.background && backgroundColors.includes(props.background)
-        ? `ffe-grid__col--bg-${props.background}`
+const backgroundClass = ({ background }) => {
+    if (!background) {
+        return null;
+    }
+
+    if (removedColours.includes(background)) {
+        throw new Error(
+            `Support for the ${background} background has been removed, please see the CHANGELOG`,
+        );
+    }
+
+    return backgroundColors.includes(background)
```

Comment:
Perhaps you can throw an error explaining to the user that an invalid color was provided here as well? This way, it's easier to spot the error!

Reply:
I've already added that..

Reply:
You've added it for removed colors - but if I passed in the string `blue-fale` instead of `blue-pale`, I'd just get a white background without any indication where the error was.

Reply:
The propTypes check already warn of that, having two errors thrown seem utterly pointless. I can see the case for doing both for a colour that was suddenly removed from the supported list but I simply can't see the point of doing it for every unsupported string of text.

Reply:
Ah didn't think of the prop types. Good point.